src/styles/blocks/page.scss

```diff
@@ -0,0 +1,6 @@
+.page {
+  font-family: $PrimaryFont;
+  color: $ColorMainAccent;
+  font-weight: $FontRegular;
```

Comment:
The variable `$FontRegular` should be defined in the `_variables.scss` file as per the code style rules.